### DISCUSSION TITLE
Differential Regional Prompter issues

### DISCUSSION THREAD

**Discussion by cololy** [hako-mikan/sd-webui-regional-prompter#367] [Ideas]
Thank you for making new great feature, Differential Regional Prompter.
I note some issues about it.

1.Hires
I use tutorial prompt about "closed eyes", and it works fine when hires off, but it does not work well when hires on.
Hires off: Closed eyes made well.
Hires on (Denoising strength 0.2): Closed eyes made. However, part of open eyes at the edge of eyes stand out than hires off.
Hires on (Denoising strength 0.6): Closed eyes not made. Eyes half open.

2.Batch Count
When batch count>1, the generated images become almost same as the image made in the first batch.

3.Lora
With this function, base prompt seems to be used twice, including lora.
So lora effect doubles, and causes unintended effects.

4.Multiple region
I want to change multiple region in same frame. (e.g. change eyes and change hands, simultaneously)
How can I write it in the schedule prompt?

**Comment by navarisun1982** [comment]
I'm interested to know what this option in scripts is (Differential Regional Prompter), but i see no guide on the main plugin documentation ?

**Reply by hako-mikan** [reply]
https://github.com/hako-mikan/sd-webui-regional-prompter/blob/main/differential_en.md
